modules/installation-cloudformation-worker.adoc

```diff
@@ -17,3 +16,24 @@ that you need for your {product-title} cluster.
 include::https://raw.githubusercontent.com/openshift/installer/release-4.18/upi/aws/cloudformation/06_cluster_worker_node.yaml[]
 ----
 ====
+[IMPORTANT]
+====
+When you use the CloudFormation template for the master machines, it provisions all three master machines with a single stack. When you use the CloudFormation template to deploy the worker machines, however, you must create the number of stacks based on your requirements, that is, the number you defined in the `install-config.yaml` file. Each stack is provisioned one time only for each machine. To provision a new worker machine, you must change the stack name.
```

Comment:
```suggestion
When you use the CloudFormation template for the master machines, it provisions all three master machines with a single stack. However, when you use the CloudFormation template to deploy the worker machines, you must create the number of stacks based on the number that you defined in the `install-config.yaml` file. Each stack is provisioned one time only for each machine. To provision a new worker machine, you must change the stack name.
```
Trying to suggest something that makes the sentence a bit less long and wordy, but I don't know if it's really improved. Use if it's useful.